Doc/library/enum.rst

```diff
@@ -847,7 +847,7 @@ Utilities and Decorators
 .. decorator:: unique
 
    A :keyword:`class` decorator specifically for enumerations.  It searches an
-   enumeration's :attr:`__members__`, gathering any aliases it finds; if any are
+   enumeration's :attr:`!__members__`, gathering any aliases it finds; if any are
```

Comment:
```suggestion
   enumeration's :attr:`~EnumType.__members__`, gathering any aliases it finds; if any are
```

Same here—unless @ethanfurman decides this shouldn't be explicitly documented, then this warning is valid and should stay.

Reply:
`__members__` should be documented (and used to be).

Reply:
Or maybe I just thought this *was* documenting it. *sigh*